packages/import-bundle/src/index.js

```diff
@@ -124,6 +146,16 @@ export async function importBundle(bundle, options = {}, powers = {}) {
   }
 }
 
+/**
+ * A utility function for producing test bundles, which are not seerializable
+ * as JSON or passable.
+ * @param {Record<PropertyKey, unknown>} exports
+ */
+export const bundleTestExports = exports => ({
```

Comment:
We should probably enforce string only data properties here too to catch mistakes early.